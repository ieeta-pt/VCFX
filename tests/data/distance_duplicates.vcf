##fileformat=VCFv4.2
##source=VCFX
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr1	10000	rs1	A	G	100	PASS	.
chr1	10000	rs2	C	T	90	PASS	.
chr1	20000	rs3	G	A	80	PASS	.
chr1	20000	rs4	T	C	70	PASS	.
chr2	15000	rs5	G	T	60	PASS	.
chr2	15000	rs6	A	C	50	PASS	.
