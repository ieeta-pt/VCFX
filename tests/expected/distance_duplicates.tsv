CHROM	POS	PREV_POS	DISTANCE
chr1	10000	NA	NA
chr1	10000	10000	0
chr1	20000	10000	10000
chr1	20000	20000	0
chr2	15000	NA	NA
chr2	15000	15000	0
